test/main.test.js

```diff
@@ -182,5 +177,12 @@ describe('GotScraping', () => {
                 expect(responseProxy.httpVersion).toBe('2.0');
             }
         });
+
+        test('should support tls 1.2', async () => {
+            const url = 'https://tls-v1-2.badssl.com:1012/';
+
+            const response = await gotScraping.get(url);
+            expect(response.statusCode).toBe(200);
+        });
```

Comment:
We should also have a test for tls 1.3 to make sure that by enabling 1.2 we have not killed 1.3